c2corg_ui/static/js/whatsnew.js

```diff
@@ -0,0 +1,94 @@
+goog.provide('app.WhatsnewController');
+goog.provide('app.WhatsnewDirective');
+
+goog.require('app');
+
+
+/**
+ * Directive managing the news.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.WhatsnewDirective = function() {
+  return {
+    restrict: 'AE',
+    scope: true,
+    controller: 'appWhatsnewController as wCtrl',
+    bindToController: true
+  };
+};
+
+app.module.directive('appWhatsnew', app.WhatsnewDirective);
+
+/**
+ * @param {!angular.Scope} $scope Scope.
+ * @param {angular.$http} $http
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {app.Alerts} appAlerts
+ * @constructor
+ * @struct
+ * @ngInject
+ * @suppress {checkTypes}
+ */
+app.WhatsnewController = function($scope, $http, $compile, appAlerts, NgTableParams) {
+
+  /**
+   * @type {app.Alerts}
+   * @private
+   */
+  this.alerts_ = appAlerts;
+
+  /**
+   *
+   * @type {Array}
+   * @export
+   */
+  this.columns = [
+    {field: 'version_id', title: 'version_id', show: true},
```

Comment:
To avoid that the Closure Compiler might rename the property names, you can either provide a definition in the externs file or use `'file': 'version_id'`.